samples/CONFIGURATION.md

```diff
@@ -155,3 +155,17 @@ defaultServiceInCaseOfException:
 
 There is an example for POST requests. To add default headers to other requests in application.yml, define it in application.yml, also in
 ApplicationProperties class the same way as HttpPost is defined and use addHeaders method from HttpQueryArgs to add these default headers to the request.
+
+
+### Internal requests
+
+Some DSL services can be exposed to to internal requests as REST endpoints.
+These can be accessed only from IPs that are specified in `allowedFiletypes` configuration block:
```

Comment:
Specified in `allowedFiletypes` or `internalRequests`?

Reply:
Fixed typo.